### PR TITLE
Feature/repsettings refactoring

### PR DESCRIPTION
**Pull request by RainbowDashLabs**
Replace the several subcommands of the repsettings with a single menu which allows to change the states.

Review blocked by #286 and #284

### CODE REVIEW

File: src/main/resources/locale.properties [hunk 1]
```diff
@@ -305,3 +300,6 @@ words.refMessage=
 words.reputation=
 words.role=
 words.unknown=
+command.repSettings.embed.choose=
```
Comment (Taucher2003) [inline src/main/resources/locale.properties:303]: Could we get this sorted correctly?
(Applies to all locale files)

File: src/main/java/de/chojo/repbot/ReputationBot.java [hunk 2]
```diff
@@ -259,6 +260,7 @@ private void initBot() {
                     }
                     log.error(LogNotify.NOTIFY_ADMIN, "Command execution of {} failed\n{}", context.command().meta().name(), context.args(), throwable);
                 })
+                .withMenuService(Consumers.empty())
```
Comment (Taucher2003) [inline src/main/java/de/chojo/repbot/ReputationBot.java:263]: Why do we need an empty Consumer here?

Comment (RainbowDashLabs) [reply src/main/java/de/chojo/repbot/ReputationBot.java]: There was no default method without an argument for adding services. I added one and used it.

File: src/main/resources/locale.properties (hunk 1 above)
Comment (RainbowDashLabs) [reply src/main/resources/locale.properties]: Sorted

**Review by Taucher2003 — APPROVED** [review]

**Pull request merged**